frontend/hub/approvals/Approvals.tsx

```diff
@@ -1,33 +1,30 @@
 import { useTranslation } from 'react-i18next';
 import { PageHeader, PageLayout, PageTable } from '../../../framework';
 import { useHubView } from '../useHubView';
-import { Approval } from './Approval';
-import { useApprovalActions } from './hooks/useApprovalActions';
+import { CollectionVersionSearch } from './Approval';
 import { useApprovalFilters } from './hooks/useApprovalFilters';
-import { useApprovalsActions } from './hooks/useApprovalsActions';
 import { useApprovalsColumns } from './hooks/useApprovalsColumns';
-import { idKeyFn, hubAPI } from '../api';
+import { hubAPI } from '../api';
 
 export function Approvals() {
   const { t } = useTranslation();
   const toolbarFilters = useApprovalFilters();
   const tableColumns = useApprovalsColumns();
-  const view = useHubView<Approval>({
-    url: hubAPI`/_ui/v1/collection-versions/`,
-    keyFn: idKeyFn,
-    toolbarFilters,
+  const view = useHubView<CollectionVersionSearch>({
+    url: hubAPI`/v3/plugin/ansible/search/collection-versions/`,
+    keyFn: (item) => item.collection_version.pulp_href + ':' + item.repository.name,
     tableColumns,
+    toolbarFilters,
+    sortKey: 'order_by',
+    defaultFilters: { status: [encodeURI('pipeline=staging')] },
```

Comment:
no encodeURI here, this needs to happen inside useHubView (and does)

Reply:
Yes, it works because encodeURI does returns original string :-D I know because I deleted it elsewhere, but forgot to delete this here. For partial URL encoding, I should use encodeURIComponent. But you are right, framework does this, so I will delete it.